docker/artemis/config/prod.env

```diff
@@ -1,7 +1,7 @@
-SPRING_PROFILES_ACTIVE="artemis,scheduling,athene,prod,docker"
+SPRING_PROFILES_ACTIVE="artemis,scheduling,athena,prod,docker"
 
 # Secret Overrides
-ARTEMIS_ATHENE_BASE64SECRET="YWVuaXF1YWRpNWNlaXJpNmFlbTZkb283dXphaVF1b29oM3J1MWNoYWlyNHRoZWUzb2huZ2FpM211bGVlM0VpcAo="
+ARTEMIS_ATHENA_BASE64SECRET="YWVuaXF1YWRpNWNlaXJpNmFlbTZkb283dXphaVF1b29oM3J1MWNoYWlyNHRoZWUzb2huZ2FpM211bGVlM0VpcAo="
```

Comment:
Does this need changing? I think you renamed it.

Reply:
You are right 👍